test-projects/test-deploy/build.sbt

```diff
@@ -1,11 +1,5 @@
-val test = project.in(file(".")).enablePlugins(DeployPlugin)
+// Tester project for basic services.
+val service = project.in(file("service")).enablePlugins(DeployPlugin)
 
-val printEnv = TaskKey[Unit]("printEnv", "Print the deploy environment")
-
-printEnv := {
-  val s = streams.value
-  val env = deployEnvironment.value
-  s.log.info(s"Environment: $env")
-}
-
-compile in Compile <<= (compile in Compile).dependsOn(printEnv)
+// Tester project for services with a node build.
+val nodeapp = project.in(file("nodeapp")).enablePlugins(WebappPlugin)
```

Comment:
This was purely because I found `webapp/webapp` unappealing to look at. I can switch back (there's not really a good reason for the name).